privacy-on-beam/pbeam/sum_test.go

```diff
@@ -320,6 +368,60 @@ func TestSumPerKeyAddsNoiseInt(t *testing.T) {
 	}
 }
 
+// Checks that SumPerKey with partitions adds noise to its output with int values. The logic
+// mirrors TestDistinctPrivacyIDAddsNoise.
+func TestSumPerKeyWithPartitionsAddsNoiseInt(t *testing.T) {
+	for _, tc := range []struct {
+		name      string
+		noiseKind NoiseKind
+		// Differential privacy params used.
+		epsilon float64
+		delta   float64
+	}{
+		// Epsilon and delta are not split because partitions are specified. All of them are used for the noise.
+		{
+			name:      "Gaussian",
+			noiseKind: GaussianNoise{},
+			epsilon:   1,    
+			delta:     0.005, 
+		},
+		{
+			name:      "Laplace",
+			noiseKind: LaplaceNoise{},
+			epsilon:   0.1, 
+			delta:     0, // It is 0 because partitions are specified and we are using Laplace noise.
+		},
+	} {
+		// We have 1 partition. So, to get an overall flakiness of 10⁻²³,
+		// we need to have each partition pass with 1-10⁻²³ probability (k=23).
+		epsilonNoise, deltaNoise := tc.epsilon, tc.delta
+		k := 23.0
+		l0Sensitivity, lInfSensitivity := 1.0, 1.0
+		l1Sensitivity := l0Sensitivity * lInfSensitivity
+		tolerance := complementaryLaplaceTolerance(k, l1Sensitivity, epsilonNoise)
+		if tc.noiseKind == gaussianNoise {
+			deltaNoise = tc.delta
```

Comment:
We need to remove this line for all such tests. It doesn't do anything.

Reply:
It was there before.

Reply:
Gotcha